deploy/template.yaml

```diff
@@ -174,6 +179,29 @@ Resources:
                           Fn::ImportValue:
                             Fn::Sub: "${VpcStackName}-ProtectedSubnetIdB"
 
+  ResultsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: Private
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
```

Comment:
What data is going into this bucket?
Should it be controlled with a KMS key instead to have tighter control of the data and what can decrypt/encrypt it?

Reply:
It is for storage of test result flat files